sn07.yml

```diff
@@ -208,6 +209,10 @@
         galaxy_fetch_dependencies: true
         galaxy_build_client: true
 
+    - role: galaxyproject.tiaas2
+      vars:
+        tiaas_virtualenv_command: "{{ galaxy_virtualenv_command }}"
```

Comment:
Itsn't this installing TIaaS into the virtual env of Galaxy?

Reply:
No, `galaxy_virtualenv_command ` contains only the path to the `virtualenv` command present in a conda environment. TIaaS env is created as usual inside `/opt/tiaas2`